src/main/java/subway/line/Line.java

```diff
@@ -0,0 +1,163 @@
+    private Long id;
+
+    @Column(length = 20, nullable = false)
+    private String name;
+
+    @Column(length = 20, nullable = false)
+    private String color;
+
+    @Column(nullable = false)
+    private Long upStationId;
+
+    @Column(nullable = false)
+    private Long downStationId;
+
+    @Column(nullable = false)
+    private Long distance;
+
+    @OneToMany(mappedBy = "line",fetch = FetchType.LAZY, orphanRemoval = true, cascade = CascadeType.PERSIST)
+    @OrderColumn(name = "section_order")
+    private List<Section>  sections = new ArrayList<>();
+
+    public Line() {
+    }
+
+    public Line(Builder builder) {
+        this.name = builder.name;
+        this.color = builder.color;
+        this.upStationId = builder.upStationId;
+        this.downStationId = builder.downStationId;
+        this.distance = builder.distance;
+    }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static class Builder {
+        private String name;
+        private String color;
+        private Long upStationId;
+        private Long downStationId;
+        private Long distance;
+
+
+
+        public Builder name(String name) {
+            this.name = name;
+            return this;
+        }
+
+        public Builder color(String color) {
+            this.color = color;
+            return this;
+        }
+
+        public Builder upStationId(Long upStationId) {
+            this.upStationId = upStationId;
+            return this;
+        }
+
+        public Builder downStationId(Long downStationId) {
+            this.downStationId = downStationId;
+            return this;
+        }
+
+        public Builder distance(Long distance) {
+            this.distance = distance;
+            return this;
+        }
+
+        public Line build() {
+            return new Line(this);
+        }
+
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Long getUpStationId() {
+        return upStationId;
+    }
+
+    public Long getDownStationId() {
+        return downStationId;
+    }
+
+    public Long getDistance() {
+        return distance;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void changeName(String name) {
+        this.name = name;
+    }
+
+    public void changeColor(String color) {
+        this.color = color;
+    }
+
+    public void addSection(Section section) {
+        validateCreateSection(section);
+        sections.add(section);
+    }
+
+    public void deleteSection(Long stationId) {
+        validateDeleteSection(stationId);
+        sections.remove(sections.size() - 1);
+    }
+
+    private void validateCreateSection(Section creatingSection) {
+        // 하행 종점이 같은 것이 존재하면 x
+        boolean isExist = sections.stream().anyMatch(section -> section.getDownStationId() == creatingSection.getDownStationId());
+        if(isExist) {
```

Comment:
- `isExist`라는 변수 이름만으로는 위의 주석이 없다면 무엇이 존재하는지 이해하기 어려울 것 같아요
조금 더 구체적인 이름으로 변경해보는 건 어떨까요?
- 전체적으로 코드 포맷팅을 한 번 점검해주시면 좋을 것 같습니다
```suggestion
        if (isExist) {
```